src/App.tsx

```diff
@@ -1,12 +1,57 @@
+import { useState } from 'react';
+import './App.css';
+
+// components
 import { AppHeader } from './components/AppHeader';
 import { AppLayout } from './components/AppLayout';
+import NewCard from './components/Card/NewCard';
+import CardsList from './components/Card/CardsList';
 
-import './App.css';
+interface CardData {
+  firstPage: string;
+  secondPage: string;
+}
 
 function App() {
+  const [cards, setCards] = useState<
+    { firstPage: string; secondPage: string }[]
```

Comment:
**nitpick:** You can use `CardData` here instead of duplicating

Reply:
Seems obvious,  I didn't notice it.